src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -203,8 +218,11 @@
                 {%- set label = name|humanize -%}
             {%- endif -%}
         {%- endif -%}
+
+        {{ widget|raw }}
```

Comment:
this should be removed once the new bs4 check/radio structure (where `input` is `label`'s sibling, not descendant) is applied

Reply:
Exact, I don't know what, but in my mind, this part didn't changed. I fix it next week, I haven't the good PC with me.

Thanks @vudaltsov and @MaPePeR (sorry to see it so late MaPePeR)

Then if it's work I can put errors in the label too @Nyholm.

Reply:
Looking at the current state of this PR: 

<img width="304" alt="screen shot 2018-02-10 at 12 19 45" src="https://user-images.githubusercontent.com/1275206/36061625-cbcf33aa-0e5c-11e8-9c86-463424baebaf.png">

```html
<fieldset class="form-group">
    <legend class="col-form-label required">Car manufacture
      <div class="invalid-feedback d-block">
        <ul class="list-unstyled mb-0">
          <li>This value should not be blank.</li>
        </ul>
      </div>
    </legend>
    <div id="super_form_carManufacture">
      <div class="form-check">
        <input id="super_form_carManufacture_0" name="super_form[carManufacture]" required="required" class="form-check-input is-invalid" value="volvo" type="radio">
        <label class="form-check-label required" for="super_form_carManufacture_0">Volvo</label>
      </div>
      <div class="form-check">
        <input id="super_form_carManufacture_1" name="super_form[carManufacture]" required="required" class="form-check-input is-invalid" value="saab" type="radio">
        <label class="form-check-label required" for="super_form_carManufacture_1">Saab</label>
      </div>
      <div class="form-check">
        <input id="super_form_carManufacture_2" name="super_form[carManufacture]" required="required" class="form-check-input is-invalid" value="koenigsegg" type="radio">
        <label class="form-check-label required" for="super_form_carManufacture_2">Koenigsegg</label>
      </div>
    </div>
</fieldset>

```
@vudaltsov: As I can see, the `<label>` and the `<input>` are already siblings. 

@mpoit: These errors looks great.

Reply:
@Nyholm , they are siblings in terms of output markup, but not siblings in terms of twig blocks. With new bs4 structure that hack with `{{ widget|raw }}` used for bs3 can be easily replaced by standard `row->(label+widget+errors)` logic.

Reply:
so the output is correct and should stay the same, but the blocks' logic should become cleaner and extensible same as for other types.

Reply:
Apparently, this is done to avoid to diaplay twice the label, I've try @vudaltsov and @MaPePeR proposition, but then Twig display the label twice: call form_label and checkbox_radio_label.

I quickly look at how to correct it properly, but if I don't find it I would leave it like that.

Reply:
Ah, took me some time to figure out how the output is correct without the `{widget: parent()}` being changed. You just moved the `{{ widget|raw }}` blocks and the `widget`-Attribute is needed to determine the correct class attributes for the label?

Reply:
We use the widget in this block to decide if we display the label or not. Because in form_row you display: label, widget and error. Now, juste label and widget.

But in a checkbox, the label is on each cheackbox, we don't want to display the label twice.